Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+@objc(TRDailyForecast) class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+
+    init?(JSON: AnyObject?) {
```

Comment:
I think that would potentially be a useful improvement in the future, when more of the app has been ported and we can drop Objective-C compatibility for the model layer. I think it's worth raising an issue to track this.